modules/_node-scripts/npmscripts/runNpmScripts.mjs

```diff
@@ -0,0 +1,129 @@
+import * as childProcess from 'child_process';
+import {constants, mkdirSync, renameSync} from 'fs';
+import * as fs from 'fs/promises';
+import resolve from 'resolve';
+
+import onExit from '../util/onExit.mjs';
+
+const DISABLE_BUILD_CONFIGS = [
+	'babel',
+	'bundler',
+	'exports',
+	'main'
+];
+
+export default async function runNpmScripts(projectNpmScriptsConfig) {
+	if (!projectNpmScriptsConfig) {
+		return;
+	}
+
+	await writeNpmScriptsConfig(projectNpmScriptsConfig);
+
+	const npmScriptsPath =
+		resolve.sync('@liferay/npm-scripts/bin/liferay-npm-scripts.js', {basedir: '.'});
+
+	const argv = [
+		'build',
+	];
+
+	if (process.argv.includes('--clean')) {
+		argv.push('--clean');
+	}
```

Comment:
We don't need this anymore I removed our cacheing layer with an earlier PR

Reply:
done